packages/web/src/pages/chat-page/components/ChatComposer.tsx

```diff
@@ -81,10 +84,17 @@ export const ChatComposer = (props: ChatComposerProps) => {
     [handleSubmit]
   )
 
+  useEffect(() => {
+    if (chatId) {
+      ref.current?.focus()
+    }
+  }, [ref, chatId])
```

Comment:
Not a big deal but just curious:

With effect deps, I tend to list them in order of likelihood to change, most likely first. The idea is that in evaluating the array each time we render, it has the probability of doing the least amount of work (assuming it short-circuits on finding the first value that has changed).

Not that there's any real performance basis for that preference, but I was curious why you do it the opposite way... 😁

Reply:
whoa first i'm hearing about this

Reply:
These are things I spend too much time thinking about 😆